weather_mv/loader_pipeline/bq.py

```diff
@@ -35,6 +35,9 @@
 DATA_IMPORT_TIME_COLUMN = 'data_import_time'
 DATA_URI_COLUMN = 'data_uri'
 DATA_FIRST_STEP = 'data_first_step'
+S2_LOCATION = 's2_location'  # st_geog_point
```

Comment:
This is misnamed. A Geography point is not the same as an S2 location. 
See, for example, [these docs](https://cloud.google.com/bigquery/docs/reference/standard-sql/geography_functions#s2_cellidfrompoint). From a Point, you can convert to a S2 location (an ID).

Reply:
The column_name is changed to geo_location.